developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -78,28 +100,184 @@ You can request additional properties by sending a request body with the :code:`
 	  </d:prop>
 	</d:propfind>
 
-The following properties are supported:
-
-- :code:`{DAV:}getlastmodified`
-- :code:`{DAV:}getetag`
-- :code:`{DAV:}getcontenttype`
-- :code:`{DAV:}resourcetype`
-- :code:`{DAV:}getcontentlength`
-- :code:`{http://owncloud.org/ns}id` The fileid namespaced by the instance id, globally unique
-- :code:`{http://owncloud.org/ns}fileid` The unique id for the file within the instance
-- :code:`{http://owncloud.org/ns}favorite`
-- :code:`{http://owncloud.org/ns}comments-href`
-- :code:`{http://owncloud.org/ns}comments-count`
-- :code:`{http://owncloud.org/ns}comments-unread`
-- :code:`{http://owncloud.org/ns}owner-id` The user id of the owner of a shared file
-- :code:`{http://owncloud.org/ns}owner-display-name` The display name of the owner of a shared file
-- :code:`{http://owncloud.org/ns}share-types`
-- :code:`{http://owncloud.org/ns}checksums`
-- :code:`{http://nextcloud.org/ns}has-preview`
-- :code:`{http://owncloud.org/ns}size` Unlike :code:`getcontentlength`, this property also works for folders reporting the size of everything in the folder.
-- :code:`{http://nextcloud.org/ns}rich-workspace` this property is provided by the text app
-- :code:`{http://nextcloud.org/ns}contained-folder-count` The number of folders directly contained in the folder (not recursively)
-- :code:`{http://nextcloud.org/ns}contained-file-count` The number of files directly contained in the folder (not recursively)
+A note about namespaces
+^^^^^^^^^^^^^^^^^^^^^^^
+
+When building the body of your DAV request, you will request properties that are available under specific namespace.
+It is usual to declare shortcuts for those namespace in the ``d:propfind`` element of the body.
+
+Here is the list of available namespace:
+
+=========================================  ========
+                Namespace                  Shortcut
```

Comment:
Those are officially called namespace URI and namespace prefix, we should use the same names.